src/applications/user-testing/disability-benefits-all-claims-conditions-chapter/content/conditions.jsx

```diff
@@ -26,19 +31,88 @@ export const NewConditionDescription = () => (
   </>
 );
 
-const createCauseFollowUpDescriptions = item => {
-  const cause = item?.cause;
+export const SecondaryEnhancedNotListedAlert = () => {
+  const formData = useSelector(state => state.form.data);
+  const currentIndex = getArrayIndexFromPathName();
+  const newCondition = formData?.[ARRAY_PATH]?.[currentIndex]?.newCondition;
+  const capitalNewCondition =
+    newCondition.charAt(0).toUpperCase() + newCondition.slice(1);
+  const demoLabel = window.location.pathname
+    .split('conditions-')[1]
+    ?.split('/')[0];
+  const targetIndex = formData?.[ARRAY_PATH]?.length;
+
+  return (
+    <va-alert status="warning">
+      <p>
+        Tell us the service-connected disability or condition that caused{' '}
+        {newCondition}. You may add it now and your progress will be saved.{' '}
+        {capitalNewCondition} will be marked as incomplete until that condition
+        is added.
+      </p>
+      <Link
+        to={`conditions-${demoLabel}/${targetIndex}/new-condition?add=true`}
+      >
+        Add new condition now
+      </Link>
+    </va-alert>
+  );
+};
+
+export const SecondaryEnhancedOptionsConflictingAlert = () => (
+  <va-alert status="error">
```

Comment:
Haven't tested this yet but if we run into focus issues we can always try adding role="alert" as well https://design.va.gov/components/alert/#assign-an-appropriate-aria-role